packages/flutter/lib/src/cupertino/split_form_row.dart

```diff
@@ -0,0 +1,199 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/widgets.dart';
+
+import 'colors.dart';
+import 'theme.dart';
+
+export 'split_form_row.dart';
+
+/// Creates a standard split row, with a helper text to one side and a child
+/// widget to the other.
+///
+/// A [leadingText] is required. This string is displayed in a [Text] widget
```

Comment:
same idea here. Would be pretty neat if this was just a widget (at which point it could be just `label` or something since it doesn't have to be a text). You can still control its position and styling via its ancestor widgets. Then people can put icons here which wouldn't be too crazy.